kas-fleet-manager/deploy-kas-fleet-manager.sh

```diff
@@ -123,9 +123,13 @@ deploy_kasfleetmanager() {
     echo "OSD_IDP_MAS_SSO_CLIENT_SECRET='${MAS_SSO_CLIENT_SECRET}'" >> ${SECRET_PARAMS}
   fi
 
-  if [ -n "${KAS_FLEET_MANAGER_SECRETS_TEMPLATE_PARAMS:-}" ] && [ -x "${KAS_FLEET_MANAGER_SECRETS_TEMPLATE_PARAMS}" ] ; then
-      echo "Executing ${KAS_FLEET_MANAGER_SECRETS_TEMPLATE_PARAMS} to generate user-supplied secrets-template.yml parameters"
-      ${KAS_FLEET_MANAGER_SECRETS_TEMPLATE_PARAMS} >> ${SECRET_PARAMS}
```

Comment:
But the script uses `set -euo pipefail` so an execution failing should have caused a failure to propagate back to the user.  Why didn't this happen?

Reply:
It's because the `KAS_FLEET_MANAGER_SECRETS_TEMPLATE_PARAMS` script never ran. 
The check for executable permission `[ -x "${KAS_FLEET_MANAGER_SECRETS_TEMPLATE_PARAMS}" ]` will be false and will jump out of the if block. So, no message will output, which causes me believe my script ran well.